cypress/integration/home.spec.ts

```diff
@@ -26,8 +26,29 @@ it('should display corresponding information', () => {
 
   cy.contains('footer', 'Created with')
   cy.contains('footer', 'by Mario')
+
+  cy.findByRole('link', { name: /mario/i }).should(
+    'have.attr',
+    'href',
+    'https://mario.dev/'
+  )
+
+  cy.findByRole('link', {
+    name: /octoclairvoyant repository on github/i,
+  }).should(
+    'have.attr',
+    'href',
+    'https://github.com/octoclairvoyant/octoclairvoyant-webapp'
+  )
+
+  cy.findByRole('img', { name: /powered by vercel logo/i })
+  // Find way to make this query below work
+  // .should('have.attr', 'src')
+  // .should('include', 'powered-by-vercel')
```

Comment:
Since it's an svg element, it might need a different way of checking the attributes.

Reply:
I can't find any reason why we are getting the base64 image instead of the URL here (might be related to how Next.js + Vercel works with SVG locally vs deploying the real app). I'd just skip the `src` check on this logo, and check it has the expected url.